src/rating/rating.ts

```diff
@@ -28,6 +40,7 @@ export interface StarTemplateContext {
 @Component({
   selector: 'ngb-rating',
   changeDetection: ChangeDetectionStrategy.OnPush,
+  host: {'(keydown)': 'handleKeyDown($event)'},
```

Comment:
Yeah, I'm still uncertain about this one, but the [wai-aria slider demo](https://www.w3.org/TR/wai-aria-practices-1.1/examples/slider/slider-2.html) and bootstrap js plugin use keydown apparently.

Plus from what I understand it's quite common to have this behaviour for navigation keys, so I'll leave it as is for now.